test/lib/doorkeeper_auth_test.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+class TestingController < ActionController::Base
```

Comment:
this will be global, so needs to be named something more unique like DoorkeeperAuthTestingController